kubetest/README.md

```diff
@@ -177,3 +180,17 @@ skew directory, potentially to upgrade/downgrade kubernetes to another version.
 [kubekins-e2e]: /jenkins/e2e-image
 [kubekins-e2e-prow]: /images/e2e-prow
 [prow]: /prow
+
+## Example Commands
+
+A simple example is:<br>
+`kubetest --up --check-version-skew=false --extract=v1.8.0-beta.1 --extract=v1.7.5 --upgrade_args=--ginkgo.focus="Feature:MasterUpgrade" --gcp-project=google.com:some-project -v`
+
+The command runs all (and only) upgrade tests tagged with `Feature:MasterUpgrade` label on GCE, upgrading the cluster from `v1.7.5` to `v1.8.0-beta.1`. Note that order of the 2 `extract` flags matters: `--extract=v2 --extract=v1` means upgrading from v1 to v2. The command does not run other e2e tests after completing the upgrade tests. If you want to run the e2e tests, specify also `--test` and `--test_args` flags.
+
+In a real test environment, the command can be longer with more parameters:<br>
+`kubetest -v --dump=/workspace/_artifacts --gcp-service-account=/etc/service-account/service-account.json --up --down --test --cluster=bootstrap-e2e --gcp-network=bootstrap-e2e --check-leaked-resources --check-version-skew=false --extract=ci/k8s-stable1 --extract=ci/k8s-stable2 --gcp-node-image=gci --gcp-zone=us-central1-f --provider=gce --test_args=--kubectl-path=path/to/kubectl.sh --minStartupPods=8 --timeout=900m --upgrade_args=--ginkgo.focus=\\[Feature:MasterUpgrade\\] --upgrade-target=ci/latest-1.6`
+
+The command run both upgrade and e2e tests on GCE at zone `us-central1-f`.
```

Comment:
yeah this long example is not that useful, unless you want to state exactly what each flag is doing.

Reply:
i will remove this.